.github/workflows/joss-paper.yml

```diff
@@ -0,0 +1,26 @@
+name: joss-paper
+on: [push]
+
+# taken from https://github.com/marketplace/actions/open-journals-pdf-generator
+
+jobs:
+  compile:
+    runs-on: ubuntu-latest
+    name: Paper Draft
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Build draft PDF
+        uses: openjournals/openjournals-draft-action@master
+        with:
+          journal: joss
+          # This should be the path to the paper within your repo.
+          paper-path: Documentation/joss/paper.md
+      - name: Upload
+        uses: actions/upload-artifact@v1
```

Comment:
Where exactly is the uploaded PDF stored?

Reply:
- see `Checks` above 
- look at the summary of the last check
- find `Artifacts` at the bottom